### DISCUSSION TITLE
blog/how-to-give-a-great-technical-presentation

### DISCUSSION THREAD

**Discussion by giscus** [cesarsotovalero/cesarsotovalero.github.io#69] [Announcements]
# blog/how-to-give-a-great-technical-presentation

I believe that the secret to delivering a great technical presentation is feeling in control. This enables the confidence to communicate effectively and it also helps to be perceived as an expert. In this article, I share the tips and tricks I’ve learned from my own flaws, and also from the best speakers.

https://www.cesarsotovalero.net/blog/how-to-give-a-great-technical-presentation.html

**Comment by martisak** [comment]
These are some great tips! 

For the section on how to start, I agree with Patrick Winston that early jokes don't work well because people are putting away their laptops. For the same reason, I don't like to start with saying my name either. Actually, I hate when a presentation starts with "My name is Martin, and I am going to talk about my research" - it's boring, and nobody will remember your name because they we're fiddling with their laptops.

The section on practice is extremely important. It reminds me of something a friend told me about a course in presentation techniques. The presenter came in and played the trumpet - extremely poorly - and said "This is why you should practice your speech.". I like to record myself, or actually, I don't like it, but I find it helpful :).

A resource I found helpful was this talk ["How to give a great research talk" by Simon Peyton Jones](https://www.microsoft.com/en-us/research/academic-program/give-great-research-talk/).

Finally, for the second footnote, I agree!

**Reply by cesarsotovalero** [reply]
Hi @martisak,

You make a good point about how boring it is to start a talk by saying the speaker's name. I think it is better if someone else introduces the speaker to the audience. Otherwise, one should go straight to the point and deliver the **promise** to solve the problem, right away. 

When it comes to practice, I'm extremely lazy. I'm still looking for the magical antidote to cure my laziness. Even for my PhD defense I only practiced 3 times before the actual defense. Maybe a good way to improve in this regard is to do video recordings (as you suggest) with the goal of publishing them on YouTube later. This would add an extra incentive to the otherwise seemly useless practice sessions while improving the quality of the presentation at the same time.  
 
The talks by Symon Peyton Jones are indeed excellent. It brings me memories from my early days when I discovered this guy, especially the one on ["How to write a great research paper"](https://www.microsoft.com/en-us/research/academic-program/write-great-research-paper/) is a good resource for researchers at all stages of their career.

**Comment by orioli** [comment]
you cna convert this post into a book :)
